--- /usr/local/lib/python3.8/dist-packages/tlslite/tlsconnection.py.old	2021-05-02 05:02:43.852203830 +0000
+++ /usr/local/lib/python3.8/dist-packages/tlslite/tlsconnection.py	2021-05-02 05:04:21.908603969 +0000
@@ -3342,7 +3342,7 @@
                 cipherSuites += \
                     CipherSuite.getSrpCertSuites(settings, version)
             cipherSuites += CipherSuite.getSrpSuites(settings, version)
-        elif cert_chain:
+        if cert_chain:
             if ecGroupIntersect or ffGroupIntersect:
                 cipherSuites += CipherSuite.getTLS13Suites(settings,
                                                            version)
